code/src/utils/Emoji.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Emoji = props => (
+    <span
+        className="emoji"
+        role="img"
+        aria-label={props.label ? props.label : ""}
+        aria-hidden={props.label ? "false" : "true"}
+    >
+        {props.symbol}
+    </span>
+);
+export default Emoji;
```

Comment:
I like the approach to make a emoji-component!

Reply:
I agree with Birgit :)